<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreviewWind</class>
 <widget class="QDialog" name="PreviewWind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查看材料列表</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lb_show_size">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="CopyableTableView" name="tv_mat">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="cb_sort_option">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>不排序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>升序</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>降序</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="lb_block_count">
     <property name="accessibleName">
      <string/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLabel" name="lb_show_volume">
     <property name="accessibleName">
      <string/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="cb_show_in_stacks">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>按组显示</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pb_export_file">
     <property name="text">
      <string>导出为文件</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pb_copy_to_clipboard">
     <property name="text">
      <string>复制到剪贴板</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CopyableTableView</class>
   <extends>QTableView</extends>
   <header>CopyableTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
